server/controllers/finance/reports/debtors/annual-clients-report.js

```diff
@@ -158,7 +134,10 @@ function getTotalsFooter(fiscalYearId, currencyId, rate, hideLockedClients = 0)
       LEFT JOIN period_total pt ON dg.account_id = pt.account_id
       LEFT JOIN account ac ON ac.id = pt.account_id
       LEFT JOIN period p ON p.id = pt.period_id
-    WHERE pt.fiscal_year_id = ? ${hideLockedClients ? hiddenClientsCondition : ''};
+    WHERE pt.fiscal_year_id = ?
+      ${hideLockedClients ? hiddenClientsCondition : ''}
+      ${includeCashClients ? '' : excludeCashClientsCondition}
+    ORDER BY ac.number DESC, dg.name DESC
```

Comment:
I think you have to order data, not the totals in footer

Reply:
Whoops, good catch.